File: Entities/Medication.cs
using System.Collections.Generic;

namespace MindYourMoodWeb.Entities
{
    public class Medication : Entity
    {
        //public int Id { get; set; }
        public string MedicationName { get; set; }
        public int TotalDailyDosage { get; set; }
        public Prescription Prescription { get; set; }
        public ICollection<MedicationSpread> MedicationSpreads { get; set; }
    }
}

File: ClientApp/src/app/_model/medication.ts
import { Prescription } from './prescription';
import { MedicationSpread } from './medicationSpread';

export interface Medication {
  id: number;
  medicationName: string;
  totalDailyDosage: number;
  prescription: Prescription;
  medicationSpreads: MedicationSpread[];
}
